pkg/signup/service/signup_service.go

```diff
@@ -178,7 +180,7 @@ func (s *ServiceImpl) Signup(ctx *gin.Context) (*v1alpha1.UserSignup, error) {
 	}
 
 	username := ctx.GetString(context.UsernameKey)
-	return nil, errors2.Errorf("unable to create UserSignup [id: %s; username: %s] because there is already an active UserSignup with such ID", encodedUserID, username)
+	return nil, errors.NewBadRequest(fmt.Sprintf("unable to create UserSignup [id: %s; username: %s] because there is already an active UserSignup with such ID", encodedUserID, username))
```

Comment:
Shouldn't it be 409 Conflict instead?

Reply:
changed to conflict here: 66887ddb508818319fd84bfd085321ff902d31bc